models/src/node/crd/mod.rs

```diff
@@ -0,0 +1,131 @@
+/// A package aim to solve the Custom Resource Migrations issue.
```

Comment:
nit: I would frame it as enabling migrations to newer versions rather than as an issue:

```suggestion
/// A package aim to enable migrations to new Custom Resource Definitions.
```